lockmyresource/core.py

```diff
@@ -39,10 +42,25 @@ class User:
     @staticmethod
     def from_os() -> "User":
         try:
+            if sys.platform.startswith("win"):
```

Comment:
Need to extract the whole username detection logic (incl. the fallback os.getlogin()) to a separate module.
import userinfo
...
username = userinfo.get_display_name()
return User(username) if username else no_user

Wanna do it or should I?

Reply:
I pushed the fix.